### PR TITLE
alt text for images updated

### PR DESCRIPTION
**Pull request by jhhornn**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/tutorials/touch_event_horizon/index.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ This article is for Touch Event Horizon and A Game Related To It
 
 ## The example game
 
-![](touch-to-start.png)
+![touch to start text type in blue font color](touch-to-start.png)
```
Comment (dipikabh) [inline files/en-us/games/tutorials/touch_event_horizon/index.md:18]:
```suggestion
![Box with the text touch to start](touch-to-start.png)
```

File: files/en-us/games/tutorials/touch_event_horizon/index.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ This article is for Touch Event Horizon and A Game Related To It
...
 
-![](tapping.png)
+![two circular blue buttons placed side-by-side with each player's score above each button](tapping.png)
```
Comment (dipikabh) [inline files/en-us/games/tutorials/touch_event_horizon/index.md:20]:
```suggestion
![Two circles placed side-by-side, one bigger than the other. The size of the circle represents a player's score. The scores are written above each circle.](tapping.png)
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @jhhornn, thanks for submitting this PR! I have added some suggestions along these guidelines https://github.com/mdn/mdn-community/discussions/168#discussioncomment-3354776.

File: files/en-us/games/tutorials/touch_event_horizon/index.md (hunk 1 above)
Comment (jhhornn) [reply files/en-us/games/tutorials/touch_event_horizon/index.md:18]: Thanks so much for this

File: files/en-us/games/tutorials/touch_event_horizon/index.md (hunk 2 above)
Comment (jhhornn) [reply files/en-us/games/tutorials/touch_event_horizon/index.md:20]: Thank you for the correction

**Comment by jhhornn** [comment]
Hello

**Review by dipikabh — APPROVED** [review]
Thank you! 🎉

**Pull request merged**